ViteMaDose/Models/VaccinationCentre.swift

```diff
@@ -165,9 +165,20 @@ extension VaccinationCentre {
         else {
             return false
         }
-        return chronoDosesCount > 0
+        
+        return chronoDosesCount > Int(truncating: RemoteConfiguration.shared.chronodoseMinCount)
```

Comment:
Sorry I forgot, here you don't need to cast anymore 
```suggestion
        return chronoDosesCount > RemoteConfiguration.shared.chronodoseMinCount
```